### ISSUE TITLE
pretrain

### ISSUE THREAD

**Issue by liuchaoheshi** [HVision-NKU/Strip-R-CNN#32]
Hello author, how did you do pre-training on ImageNet? Is it the ImageNet-1k dataset? Can you tell me the specific configuration?

**Comment by YXB-NKU** [comment]
https://github.com/huggingface/pytorch-image-models , we use timm to  pretrain our model on imagenet-1k dataset, batchsize=1024, learning rate = 1e-3
